From e48c2b8f3fdc4cd0c2d89a5ea99ea4647f16d7ee Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 13 Feb 2014 21:39:50 +0000
Subject: [PATCH] Use bindtextdomain in some programs where it was missing.

Found by make syntax-check.

(cherry picked from commit 9018356a67eb644126814e4ef41555c56e884a79)
---
 builder/Makefile.am      | 3 +++
 builder/index-validate.c | 4 ++++
 2 files changed, 7 insertions(+)

diff --git a/builder/Makefile.am b/builder/Makefile.am
index 9d2dbc5..3087ac4 100644
--- a/builder/Makefile.am
+++ b/builder/Makefile.am
@@ -224,6 +224,9 @@ virt_index_validate_SOURCES = \
 	index-struct.c \
 	index-validate.c
 
+virt_index_validate_CPPFLAGS = \
+	-DLOCALEBASEDIR=\""$(datadir)/locale"\"
+
 man_MANS += virt-index-validate.1
 noinst_DATA += $(top_builddir)/html/virt-index-validate.1.html
 
diff --git a/builder/index-validate.c b/builder/index-validate.c
index d156c43..26abaa8 100644
--- a/builder/index-validate.c
+++ b/builder/index-validate.c
@@ -61,6 +61,10 @@ main (int argc, char *argv[])
   const char *input;
   struct section *sections;
 
+  setlocale (LC_ALL, "");
+  bindtextdomain (PACKAGE, LOCALEBASEDIR);
+  textdomain (PACKAGE);
+
   for (;;) {
     c = getopt_long (argc, argv, options, long_options, &option_index);
     if (c == -1) break;
-- 
1.8.5.3

